pylint/checkers/base/comparison_checker.py

```diff
@@ -243,14 +252,14 @@ def _check_logical_tautology(self, node: nodes.Compare) -> None:
             suggestion = f"{left_operand} {operator} {right_operand}"
             self.add_message("comparison-with-itself", node=node, args=(suggestion,))
 
-    def _check_two_literals_being_compared(self, node: nodes.Compare) -> None:
-        """Check if two literals are being compared; this is always a logical tautology."""
+    def _check_constants_comparison(self, node: nodes.Compare) -> None:
+        """When two constants are being compared it is always a logical tautology."""
         left_operand = node.left
-        if not isinstance(left_operand, nodes.Const):
+        if not (isinstance(left_operand, nodes.Const)):
```

Comment:
The parenthesis are not required right ?
```suggestion
        if not isinstance(left_operand, nodes.Const):
```

Reply:
The conversation is resolved but it does not seem the fix was applied ? (Same for https://github.com/PyCQA/pylint/pull/7526/files#r990597254 and https://github.com/PyCQA/pylint/pull/7526/files#r990597282)